backend/src/main/kotlin/app/ehrenamtskarte/backend/cards/webservice/schema/CardQueryService.kt

```diff
@@ -15,32 +15,22 @@ class CardQueryService {
     @Deprecated("Deprecated since May 2023 in favor of CardVerificationResultModel that return a current timestamp", ReplaceWith("verifyCardInProjectV2"))
     @GraphQLDescription("Returns whether there is a card in the given project with that hash registered for that this TOTP is currently valid and a timestamp of the last check")
     fun verifyCardInProject(project: String, card: CardVerificationModel, dfe: DataFetchingEnvironment): Boolean {
-        val context = dfe.getContext<GraphQLContext>()
-        val projectConfig = context.backendConfiguration.getProjectConfig(project)
-        val cardHash = Base64.getDecoder().decode(card.cardInfoHashBase64)
-        var verificationResult = false
-
-        if (card.codeType == CodeType.STATIC) {
-            verificationResult = card.totp == null && CardVerifier.verifyStaticCard(project, cardHash, projectConfig.timezone)
-        } else if (card.codeType == CodeType.DYNAMIC) {
-            verificationResult = card.totp != null && CardVerifier.verifyDynamicCard(project, cardHash, card.totp, projectConfig.timezone)
-        }
-        Matomo.trackVerification(context.backendConfiguration, projectConfig, context.request, dfe.field.name, cardHash, card.codeType, verificationResult)
-        return false
+        return verifyCardInProjectV2(project, card, dfe).valid
```

Comment:
Do we really need to update this or can we delete this function as it is deprecated for 1.5 years and we do not use it?

Reply:
There was a discussion about that above. I decided to refactor and keep it. Alternatively we should check if we don’t have users with the old app version.